app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/SettingsWidget.java

```diff
@@ -322,6 +346,19 @@ private void showDeveloperOptionsDialog() {
         widget.show();
     }
 
+    private void showControllerOptionsDialog() {
+        mWidgetManager.pushWorldBrightness(this, WidgetManagerDelegate.DEFAULT_DIM_BRIGHTNESS);
+        hide(REMOVE_WIDGET);
+        UIWidget widget = getChild(mControllerOptionsDialogHandle);
+        if (widget == null) {
+            widget = createChild(ControllerOptionsWidget.class, false);
+            mControllerOptionsDialogHandle = widget.getHandle();
+            widget.setDelegate(() -> onOptionsDialogDismissed());
+        }
+
+        widget.show();
+    }
+
     private void showLanguageOptionsDialog() {
```

Comment:
UI: I notice in the [UIS-16 Settings design spec](https://github.com/MozillaReality/FirefoxReality/files/2909833/UIS-16_Settings_016.pdf), there is a subtle horizontal gradient that appears when there is further content on an end of the scrollable container.

![image](https://user-images.githubusercontent.com/203725/54201956-3235af00-448c-11e9-9194-189bf25a0b93.png)

currently, (even with the scrollbar) I sometimes get confused as to whether there is content to scroll and whether  the text is truncated. a gradient would help that. admittedly, it's probably more engineering effort to implement than the benefit would be.

![image](https://user-images.githubusercontent.com/203725/54202105-86d92a00-448c-11e9-886f-49a152ae8b4a.png)